spec/acceptance/sensu_secrets_spec.rb

```diff
@@ -171,16 +175,24 @@
           "burst" => 200
         },
       }
+      file { '/tmp/secret':
+        ensure  => 'file',
+        content => "supersecret2\n",
+        notify  => Sensu_secrets_vault_provider['my_vault-token_file'],
+      }
       sensu_secrets_vault_provider { 'my_vault-token_file':
         ensure       => 'present',
-        address      => "https://vaultserver.example.com:8201",
-        token_file   => '/tmp/secret',
+        address      => "https://vaultserver.example.com:8200",
+        token_file   => "/tmp/secret",
```

Comment:
We should be consistent in using single quotes for non-interpolated strings.

Reply:
I only changed this particular line back to single quotes. The majority of the strings in this file and other files used for acceptance testing are using double quotes and will generate a very large diff if I start changing them all.